root/static/partials/c_instantanswer.html

```diff
@@ -0,0 +1,24 @@
+<!-- Outside is wrapped within <div class="ia..."> -->
+
+<div class="grid-6">
+  <strong><a href="https://duck.co/ia/view/{{ia.id}}" ng-click="addImg('iapage', ia.id)">{{ia.name}}</a></strong>
```

Comment:
@alohaas it looks like the data that I have is out of data--that Markdownreference one has been ghosted and shouldn't show up at all (it's not showing up here: https://duck.co/u/guiltydolphin/). Thanks!